web-console/Storage/OCS/OCS _Installation_Workflow.md

```diff
@@ -0,0 +1,76 @@
+# OCS Install Workflow
+
+### Operator Hub
+
+There will be 2 OCS operator entries in Operator Hub covering each of the following modes. 
+* Converged Mode -- Local Cluster / Hyperconverged 
+* Independent Mode -- External Cluster (Ceph Cluster is deployed externally)
+
+### Converged Mode Design
+
+Operator Hub
+![Operator Hub operators view](img/OCS-operator-view.png)
+
+Installation
+![Operator Hub operators view](img/OCS-Install-step1.png)
+
+![Operator Hub operators view](img/OCS-Install-step2.png)
+
+After the subscription process, the user will see the subscribed operator page. 
+In the installed operator view the user can see the OCS operator installed. 
+
+OCS Overview page
+![OCS-View](img/OCS-View.png)
+
+
+# OCS Installation UPI (deployed on the cloud)
+
+Clicking on "create new" (OCS Service) Will start the following installation flow: 
+### Step 1- Select Nodes
+ * Admin needs to select 3 nodes to label with “role=storage-node” to be used for the OCS cluster.
```

Comment:
Since we are only supporting dedicated storage nodes to start, we should be applying two labels:

* `node-role.kuberentes.io/storage=""`
* `node.ocs.openshift.io/cluster="<cluster_name>"`

We would need to make sure than neither `node-role.kubernetes.io/master` nor `node-role.kubernetes.io/worker` are set.

Reply:
Why does it matter if `master` or `worker` are set? If they select certain nodes we should be able to put ceph on it with our labels and ignore the other labels. Though we might warn them if they're selecting masters.